### PR TITLE
Implement version parsing

### PR DESCRIPTION
**Pull request by lukacupic**
PR for #19

I also bumped log4j's versions (both core and api) to 2.14.0 because I was having some errors with the previous (lower) versions.

### CODE REVIEW

**Comment by charlesmike** [comment]
@lukacupic thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by charlesmike** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

**Comment by lukacupic** [comment]
I wasn't sure if I should be writing tests for this one, but now that the coverage has dropped I'm guessing I should.
@amihaiemil what do you think?

File: src/main/java/com/selfxdsd/todocli/TodoFinderCli.java [hunk 1]
```diff
@@ -51,8 +62,51 @@
...
-        logger.info("Running TodoCLI v1.0 from directory '" + root + "'\n");
+        initOptions(args);
+    }
+
+    /**
+     * Initializes program options, i.e. possible command line arguments.
+     *
+     * @param args Command line arguments
+     */
+    private static void initOptions(final String[] args) {
```
Comment (amihaiemil) [inline src/main/java/com/selfxdsd/todocli/TodoFinderCli.java:73]: @lukacupic Logic in this method should probably be pulled out in a proper object which will be an Iterable of String (each option being a String). 

But we can do this in a future task. Then, we will also be able to write unit tests for it.

**Review by amihaiemil — COMMENTED** [review]
@lukacupic It's ok for a first version see my comment. Many thanks!

**Comment by amihaiemil** [comment]
@rultor merge it please

File: src/main/java/com/selfxdsd/todocli/TodoFinderCli.java (hunk 1 above)
Comment (lukacupic) [reply src/main/java/com/selfxdsd/todocli/TodoFinderCli.java:73]: Yes, as more options are added, a proper object should replace the method. Feel free to open this up as an issue if you already have some options in mind.

**Pull request merged**